DigitalLearningSolutions.Web/Helpers/SupportLinksHelper.cs

```diff
@@ -2,7 +2,8 @@
 {
     public class SupportLinksHelper
     {
-        public static string HelpUrl(string currentSystemBaseUrl) => $"{currentSystemBaseUrl}/help/Introduction.html";
+        public static string GenerateSupportTicketsIframeUrl(string currentSystemBaseUrl) => $"{currentSystemBaseUrl}/tracking/tickets?nonav=true";
+        public static string GenerateHelpUrl(string currentSystemBaseUrl) => $"{currentSystemBaseUrl}/help/Introduction.html";

```

Comment:
`Generate` is just a verb here right? Synonymous with `Get`?
I think it's a good idea to have something like that to make `HelpUrl` clearer. Could we add it to `ChangeRequestUrl` as well so that `Change` doesn't get interpreted as a verb?
I think "Get" may be a slightly better prefix than "Generate" because it's more commonplace. And with C# you're not exactly generating a new string every time because strings are [interned](https://docs.microsoft.com/en-us/dotnet/api/system.string.intern?view=net-6.0#remarks). Feel free to argue back.

Reply:
Yeah that makes sense, especially with the `Change` one needed clarification on expectation of use. That made me chuckle.